src/clr-angular/modal/modal.html

```diff
@@ -32,8 +32,8 @@
                 </div>
             </div>
             <div class="modal-ghost-wrapper">
-                <div [@ghostPageOneState]="ghostPageState" class="modal-ghost modal-ghost-1"></div>
```

Comment:
Let's put an HTML comment in here to mark this deprecation.

Reply:
👍